public/app/app-init.js

```
@@ -0,0 +1,617 @@
+function setFileLabel(name, sizeMB) {
+  const label = document.getElementById('file-label') ||
+                document.querySelector('.file-name, .drop-label');
+  if (label) label.textContent = `${name} (${sizeMB.toFixed(1)} MB)`;
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+//  ⑨ Worklet message handler (meter / RMS → UI)
+// ─────────────────────────────────────────────────────────────────────────────
+
+function handleWorkletMessage(ev) {
+  const { type, rms, frame } = ev.data;
```

ESLint

[error] 'frame' is assigned a value but never used.

(no-unused-vars)